code/modules/projectiles/projectile/energy/ebow.dm

```diff
@@ -1,16 +1,25 @@
 /obj/item/projectile/energy/bolt //ebow bolts
 	name = "bolt"
 	icon_state = "cbbolt"
-	damage = 8
+	damage = 15
 	damage_type = TOX
 	nodamage = 0
 	knockdown = 100
 	stutter = 5
-	drowsy = 50
+	var/radiation_max = 900
+	var/radiation_increase = 300
+
+/obj/item/projectile/energy/bolt/on_hit(atom/A, blocked)
+	. = ..()
+	if(isliving(A))
+		var/mob/living/M = A
+		M.radiation = min(M.radiation + radiation_increase, radiation_max)

```

Comment:
Make sure to check if the radiation does not exceed the radiation_max value so we don't cap greater amounts of radiation.

Reply:
thx